rpcs3/Emu/CPU/CPUThread.cpp

```diff
@@ -479,6 +479,8 @@ namespace cpu_counter
 
 void cpu_thread::operator()()
 {
+	thread_ctrl::scoped_priority stock_prio(0);
```

Comment:
What's going on here? Why do you explicitly need to set "normal" priority on background workers?

Reply:
this is to ensure all normal priority threads run on 10 niceness value

Reply:
It's not required, named_thread does it internally, but it doesn't hurt either and may be good for documentation purposes.